src/components/pages/Animations/FaceoffReveal.js

```diff
@@ -207,9 +217,9 @@ const FaceoffReveal = props => {
           {useCountdownStyle1.number}
         </animated.h1>
         {/* type of matchup (drawing/story) */}
-        <animated.h1 style={shiftUpStyle} className="resultsType">
-          {matchupType} Results...
-        </animated.h1>
+        <animated.div style={shiftUpStyle} className="resultsType">
+          <h1>{matchupType} Results...</h1>
```

Comment:
This should be an h2 since the header is already the h1

Reply:
In elements.scss, add
`h1, .h1 {`
so that you can use the `.h1` class to style the h2 to look like an h1